tomoScanApp/Db/tomoScan.template

```diff
@@ -199,6 +199,17 @@ record(ao, "$(P)$(R)ExposureTime")
    field(PREC,  "3")
 }
 
+record(ao, "$(P)$(R)BrightExposureTime")
```

Comment:
Everywhere else the term "FlatField" is used, not Briight.  This should be changed to FlatFieldExposureTime or if that is too long "FlatExposureTime".  Similarly it should be "DifferentFlatExposureTime".

Reply:
Actually I think it should be "DifferentFlatExposure" without Time.  The PV name is getting near the maximum length depending on the prefixes.